core/src/test/java/com/graphhopper/routing/util/CarFlagEncoderTest.java

```diff
@@ -634,9 +620,9 @@ public void testCombination() {
         way.setTag("highway", "cycleway");
         way.setTag("sac_scale", "hiking");
 
-        EncodingManager.AcceptWay map = new EncodingManager.AcceptWay();
-        assertTrue(em.acceptWay(way, map));
-        IntsRef edgeFlags = em.handleWayTags(way, map, em.createRelationFlags());
+        assertEquals(EncodingManager.Access.CAN_SKIP, encoder.getAccess(way));
+        assertNotEquals(EncodingManager.Access.CAN_SKIP, ((AbstractFlagEncoder) em.getEncoder("bike")).getAccess(way));
+        IntsRef edgeFlags = em.handleWayTags(way, em.createRelationFlags());
```

Comment:
Not sure why we use a bike encoder for this car encoder test, but I just kept the logic the same.

Reply:
Likely because initially there were a couple of bit handling bugs in EncodingManager (and/or EncodedValue) when using multiple flag encoders like overlapping ranges or similar and we needed the test against that somewhere I guess but yes those tests do not belong here.

Reply:
Ok, I'll leave it as it is then